README.md

```diff
@@ -169,26 +169,25 @@ See wiki, in particular
 - [https://github.com/sul-dlss/preservation_catalog/wiki/Audits-(basic-info)](Audits (basic info))
 - [https://github.com/sul-dlss/preservation_catalog/wiki/Audits-(how-to-run-as-needed)](Audits (how to run as needed))
 
-## Seed the Catalog 
+## Populate the Catalog
 
-Seed the Catalog with data about the Moabs on the storage roots the catalog tracks -- presumes rake db:seed already performed)
+Populate the Catalog with data about the Moabs already written to the storage roots the catalog tracks -- presumes `rake db:seed` already performed
 
-_<sub>Note: "seed" might be slightly confusing terminology here, see https://github.com/sul-dlss/preservation_catalog/issues/1154</sub>_
-
-Seeding the catalog presumes an empty or nearly empty database -- otherwise seeding will throw `druid NOT expected to exist in catalog but was found` errors for each found object.
-Seeding does more validation than regular M2C.
+Populating the catalog presumes an empty or nearly empty database -- otherwise walking the storage roots to populate it will throw
+`druid NOT expected to exist in catalog but was found` errors for each found object.  Catalog population does more validation than the
+similar `MoabToCatalogJob`/`MoabRecordService::CheckExistence`.
 
 From console:
 ```ruby
-Audit::MoabToCatalog.seed_catalog_for_all_storage_roots
+CatalogUtils.populate_catalog_for_all_storage_roots
```

Comment:
this change, and similar changes elsewhere in the README, simultaneously update a defunct class name (`Audit::MoabToCatalog` -> `CatalogUtils`) and renames a class method to be less confusing (switching terminology from "seed" to "populate" when referring to loading up the database with `MoabRecord`s, since `seed` has a specific meaning in Rails, and since we use that Rails mechanism to load the database with config info about storage roots and cloud endpoints, which other DB records refer to).